### ISSUE TITLE
<a></a>

### ISSUE THREAD

**Issue by mml1** [NegativeKarma/cc_capstone_project#2]
You have many pages with `<a></a>`. These are empty links and should be removed. Even though you are not using these links, a screen reader will see them.
